invoicer_project/static/js/request_utils.js

```diff
@@ -187,4 +187,20 @@ export function addDeleteButtonListeners(selectorName, url) {
             }
         });
     });
-}
\ No newline at end of file
+}
+
+export function search(elements, elementDivs) {
+    let input = document.getElementById('search_bar').value;
+    input = input.toLowerCase();
+    let x = document.getElementsByClassName(elements);
```

Comment:
let's use self describe names of variables. what is x? is it coordinate?

Reply:
replace let to const

Reply:
Done